src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo : Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={classNames(
+    'TodoInfo',
+    { 'TodoInfo--completed': todo.completed },
+  )}
```

Comment:
Also, just fyi ( for your information )
You can make libs variable classname shortest
just `import cn from classnames`
```suggestion
  <article className={cn('TodoInfo', {
     'TodoInfo--completed': todo.completed 
   })}
```